action.yml

```diff
@@ -19,6 +19,11 @@ inputs:
       Each tag must be in the format: <registry>/[<namespace>/]<repository>:<tag>
     required: true
 
+  artifactory_token:
```

Comment:
We don't have access to the `secrets` context in a composite action.  Having to pass it in as a required field doesn't feel great, but I can't think of another option.  Is there a better way to handle this?

Reply:
Hm, I didn't realise this change would require authentication. Because this is a public action, it needs to be able to run on GitHub-hosted runners, which don't have access to Vault. Therefore it would mean we'd need to add a GitHub secret to the org, or each project needing it.

Is there a reason we want to keep the Docker image private? Would it not be possible to push it to a public repository so we can avoid needing authentication?

Reply:
(If authentication ends up being unavoidable, passing it in this way would be fine IMO, there aren't really any other choices when using GitHub-hosted runners.)

Reply:
I've got a request in with prodsec to see if we can move where we're hosting that image.